python/packages/nisar/workflows/focus.py

```diff
@@ -1551,6 +1552,15 @@ def get_focused_sub_swaths(rawlist, out_chan, grid, orbit, doppler, dem, azres,
     return swaths
 
 
+def get_azimuth_window(cfg: Struct):
```

Comment:
Let's annotate the return type as well.

```suggestion
def get_azimuth_window(cfg: Struct) -> isce3.core.ChebyKernelF32 | None:
```